Dockerfile

```diff
@@ -7,16 +7,49 @@ RUN apt-get update && apt-get install -y \
   mediainfo \
   unzip \
   && rm -rf /var/lib/apt/lists/*
-
 RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
 RUN php composer-setup.php --install-dir=/bin --filename=composer --version=1.10.16
 RUN php -r "unlink('composer-setup.php');"
 
 # Set Timezone
 RUN echo "date.timezone = Europe/London" > /usr/local/etc/php/conf.d/timezone_set.ini
 
-COPY composer.json composer.lock /var/www/html/
+###########################################################################################
+# Run test suite
+###########################################################################################
+
+FROM base AS test
+
+# Remove the memory limit for the CLI only.
+RUN echo 'memory_limit = -1' > /usr/local/etc/php/php-cli.ini
+
+# Remove the vanilla Drupal ready to install a dev version
+RUN rm -rf ..?* .[!.]* *
+
+# Install Drupal 8.x Dev
+RUN composer create-project drupal-composer/drupal-project:8.x-dev . --stability dev --no-interaction
+
+# Update autoloads
+RUN composer dump-autoload --optimize
+
+# Install custom modules and run PHPUnit
+WORKDIR /opt/drupal/web
+
+COPY phpunit.xml core/phpunit.xml
+COPY modules/custom modules/custom
+
+RUN ../vendor/bin/phpunit -c core --testsuite unit --debug --verbose
 
+###########################################################################################
+# Create runtime image
+###########################################################################################
+
+FROM base
+
+WORKDIR /opt/drupal/web
```

Comment:
The parent Dockerfile sets `WORKDIR` to be `/opt/drupal`. I'm interested in why it needs to be different for our project?

If we do need to change it, we can use `WORKDIR web` to shift _into_ the web dir. That might reduce the chance of a future change like the one we saw recently from breaking our build.